velox/functions/prestosql/window/tests/AggregateWindowTest.cpp

```diff
@@ -131,7 +131,9 @@ TEST_F(AggregateWindowTest, variableWidthAggregate) {
 
 // Tests function with k RANGE PRECEDING (FOLLOWING) frames.
 TEST_F(AggregateWindowTest, rangeFrames) {
-  for (const auto& function : kAggregateFunctions) {
+  auto aggregateFunctions = kAggregateFunctions;
+  aggregateFunctions.push_back("array_agg(c2)");
```

Comment:
What about adding "array_agg(c2)" to kAggregateFunctions?

Reply:
@kagamiori : That's a good suggestion. I tried it but it broke many tests. So it might not be a good idea for now.

Reply:
@aditi-pandit, Got it. Thank you for trying. Do you know why other tests are broken if we add "array_agg(c2)" to kAggregateFunctions? Do the failures due to new bugs or just some inconsistency with DuckDB?